### PR TITLE
Utils/division

### PR DESCRIPTION
**Pull request by jordaniza**
Added helper utilities for ethers bigNumber division

### CODE REVIEW

File: libs/sdk-utils/multicall/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export function sdkUtilsMulticall(): string {
+    return 'sdk-utils-multicall';
+  }
+  
+  
+  /**
+   * MyType extends a `thing` that does some `stuff`
+   */
+  export interface MyType {
```
Comment (adam-arold) [inline libs/sdk-utils/multicall/src/index.ts:9]: Do we need this?

File: libs/sdk-utils/multicall/src/lib/sdk-utils-multicall.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+export function sdkUtilsMulticall(): string {
+  return 'sdk-utils-multicall';
+}
+
+
+/**
+ * MyType extends a `thing` that does some `stuff`
+ */
+interface MyType {
```
Comment (adam-arold) [inline libs/sdk-utils/multicall/src/lib/sdk-utils-multicall.ts:9]: Same as above.

File: libs/sdk-utils/multicall/src/index.ts (hunk 1 above)
Comment (jordaniza) [reply libs/sdk-utils/multicall/src/index.ts:9]: will rebase hold on

Comment (jordaniza) [reply libs/sdk-utils/multicall/src/index.ts:9]: No we can delete, it's placeholder that assumes the init branch has been merged

File: libs/shared/helpers/src/lib/fractionalDivide.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { BigNumber, BigNumberish, ethers } from 'ethers'
```
Comment (jordaniza) [inline libs/shared/helpers/src/lib/fractionalDivide.ts:1]: @Alexintosh could you please sanity check this file

**Review by adam-arold — APPROVED** [review]
Ok, looks good then @jordaniza .

**Comment by gbonumore** [comment]
@jordaniza as we discussed, let's remove the yarn.lock in favor of package-lock.json

**Review by gbonumore — APPROVED** [review]

**Pull request merged**